### ISSUE TITLE
CIFAR 10 ipc 10 Hyperparameters

### ISSUE THREAD

**Issue by WeizhiGao** [GeorgeCazenavette/mtt-distillation#31]
Hi George,

Thanks for your amazing work!

I am testing your codes in the setting of CIFAR10 ipc 10. I adopt the hyperparameters list in the readme page. However, gradient explosion ocurrs on parameter 'syn_lr'. By changing the learning rate (pxiel) to 100 or 1000, I can get a stable result. I wonder if there is a typo in the table.

Your anwer will be really helpful!

Much Thanks.

**Comment by sonbk220** [comment]
I am having the same problem.

**Comment by GeorgeCazenavette** [comment]
Hi all, apologies for the late reply.

Yes, this is definitely a typo in the table, sorry about that.

I can try to dig up the old experiment logs and confirm the correct value we used if you're curious.

**Comment by Thats2Great** [comment]
I have the same problem

**Comment by alchowiw** [comment]
> Hi all, apologies for the late reply.大家好，很抱歉回复晚了。
> 
> Yes, this is definitely a typo in the table, sorry about that.是的，这绝对是表格中的拼写错误，很抱歉。
> 
> I can try to dig up the old experiment logs and confirm the correct value we used if you're curious.如果你好奇的话，我可以尝试挖掘旧的实验日志并确认我们使用的正确值。

I would really like to know the correct parameter settings！ can you show it?
